providers/linux/os.go

```diff
@@ -229,7 +228,7 @@ func findDistribRelease(baseDir string) (*types.OSInfo, error) {
 		}
 		return osInfo, err
 	}
-	return nil, fmt.Errorf("no valid /etc/<distrib>-release file found: %w", &multierror.MultiError{Errors: errs})
+	return nil, fmt.Errorf("no valid /etc/<distrib>-release file found: %w", errors.Join(errs...))
```

Comment:
To clarify that this is an error path, perhaps make the return at L229 be `return osInfo, nil`?

Reply:
Yeah, that looks better. 4b71d5d